### PR TITLE
Convenience test packages

### PR DESCRIPTION
**Pull request by amacleay**
Add convenience `Test::Selenium::*` packages for `Selenium::*` convenience packages

Should address #250

### CODE REVIEW

**Comment by amacleay** [comment]
Ah, I see why the build is failing: I didn't add linux recordings, only Mac.  Is there a good way for me to generate recordings for different systems without getting set up on another computer?  I checked [the contribution guide](https://github.com/gempesaw/Selenium-Remote-Driver/wiki/Contribution-Guide#integration-tests) and didn't see any advice on that particular front.

**Comment by gempesaw** [comment]
>  Is there a good way for me to generate recordings for different systems without getting set up on another computer? 

No, unfortunately it is a bit of hassle, as you've surmised. I have to maintain a couple linux VMs for this purpose, I can add the other recordings :)

**Comment by amacleay** [comment]
Great, thanks very much!

File: lib/Test/Selenium/Chrome.pm [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+Test::Selenium::Chrome
+
+=head1 SYNOPSIS
+
+    my $test_driver = Test::Selenium::Chrome->new;
+    $test_driver->get_ok('https://duckduckgo.com', "Chrome can load page");
+	$test_driver->quit();
+
+=head1 DESCRITION
```
Comment (gempesaw) [inline lib/Test/Selenium/Chrome.pm]: Heh, looks like `DESCRIPTION` is misspelled in all four classes.

**Comment by gempesaw** [comment]
overall looks good, thanks much!

File: t/convenience.t [hunk 2]
```diff
@@ -18,20 +22,52 @@ my %caps = %{ $harness->base_caps };
...
+
+		my $ie = Selenium::InternetExplorer->new( %caps );
+		ok( $ie->browser_name eq 'internet_explorer', 'convenience ie is okay' );
+		$ie->quit;
+	}
+};
+
+subtest TestDriver => sub {
+	my $phantomjs = Test::Selenium::PhantomJS->new( %caps );
+	ok( $phantomjs->browser_name eq 'phantomjs', 'convenience phantomjs is okay' );
```
Comment (vkatsikaros) [inline t/convenience.t]: Both `subtest TestDriver` and `subtest Driver` `ok` messages refer to the driver objects (`Test::Selenium::<BROWSER>` or `Selenium::Chrome::<BROWSER>`) as `convenience`. 

Should `subtest TestDriver` refer to `convenience` and `subtest Driver` to something else (like `normal`)?

**Pull request closed without merging**